docs: W-7274674 (#1206)
forcedotcom/cli: releasenotes/README.md

These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.

> * NEW: Some commands (`force:org:open`, `force:org:display`, `force:user:display`, `force:source:open`) output tokens and URLs with tokens embedded in them.  They now include a warning about the risks of sharing that sensitive information. 

Reviewer: Seems like a verb is missing. Some commands (yadda) **use** output tokens...

Author: `output` is a verb but can also be a noun.  Would you rather say 'some commands **print** tokens?'

Reviewer: Doh! Words are hard. I misread it - so others might. what about "display" ? If not, we can stick with output.

> * NEW: `force:source:retrieve -m CustomField` (or the manifest equivalent using `*` for CustomField without CustomObject entries) succeeds but returns no results.  It's a confusing Metadata API quirk.  Now, when you run the command like that, it will output a warning with instructions for how to do what you probably meant to do.

Reviewer: Can we use different wording for "It's a confusing Metadata API quirk"? ...no results due to an issue with how the Metadata API interprets the command flag (I don't know what the quirk is but something like that?)

Author: it's not really an "issue" (like a problem that'll be fixed) just more a thing people don't know that doesn't work how they expect.

Can you say what the goal of different wording would be?

Reviewer: Just to reframe it so it doesn't sound like a criticism. If it's WAD albeit confusing, perhaps just say (It's just how Metadata API works but we want to provide you the results that you're looking for.) Something like that. It's totally a judgment call, of course.

> * NEW: `force:source:retrieve -m CustomField` (or the manifest equivalent using `*` for CustomField without CustomObject entries) succeeds but returns no results.  It's a confusing Metadata API quirk.  Now, when you run the command like that, it will output a warning with instructions for how to do what you probably meant to do.

Reviewer: ```suggestion
* NEW: `force:source:retrieve -m CustomField` (or the manifest equivalent using `*` for CustomField without CustomObject entries) succeeds but returns no results.  It's a confusing Metadata API quirk.  Now, the command outputs a warning with instructions for how to modify so that you see the results.
```

Author: It's not that you can't see the result, but that there is nothing retrieved/returned.  Is that how it reads?

Reviewer: Ah. ok. I was unsure what "probably mean to do" meant. What does the warning display? What are the instructions?
